data-manifest

```diff
@@ -59,11 +59,9 @@ test/cibtests/004.exp.xml
 test/cibtests/004.input
 test/cib-tests.sh
 test/cibtests/shadow.base
-test/containerized-regression-tests.sh
```

Comment:
It's more a niggle, the test/containerized-regression-tests.sh, test/docker_scripts.sh was removed in another commit.